app/resources/ephemera_folders/ephemera_folder_decorator.rb

```diff
@@ -58,8 +58,6 @@ def collections
   # TODO: Rename to decorated_ephemera_box
   def ephemera_box
     wayfinder.decorated_ephemera_box
-  rescue ArgumentError # Necessary because ability_spec.rb doesn't always save objects. TODO: Fix.
```

Comment:
I don't really understand from this comment. Maybe you could write it in an issue?

Reply:
Sure. See #4379.

Reply:
Thank you!